File: 백준/Bronze/11720. 숫자의 합/숫자의 합.cc
#include <stdio.h>

char s[101];
int main(void) {
    int n;
    scanf("%d", &n);
    scanf("%s", s);
    
    int sum_value = 0;
    for (int i = 0; i < n; i++) sum_value += (s[i] - '0');
    
    printf("%d", sum_value);
    return 0;
}

File: 백준/Bronze/11720. 숫자의 합/숫자의 합.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int n = sc.nextInt();
        sc.nextLine();
        String s = sc.nextLine();
        
        int sum_value = 0;
        for (int i = 0; i < n; i++) sum_value += (int)s.charAt(i) - '0';
        
        System.out.println(sum_value);
    }
}